gstaichi/python/export_lang.cpp

```diff
@@ -370,6 +370,8 @@ void export_lang(py::module &m) {
            [](Program *program, SNode *snode, int element_ndim, int n, int m) {
              return field_to_dlpack(program, snode, element_ndim, n, m);
            })
+      .def("_unsupported_get_num_ndarrays",
+           &Program::unsupported_get_num_ndarrays)
```

Comment:
Why is this considered unsupported?

By the way, I would recommend using readonly property instead of function for functions that are basically getters. (and read/write properties if there is any setter).

Reply:
(I tried to use `_` prefix, but clang tidy wouldnt allow me to)

Reply:
As you want, but this is not a common practice at all for python-first package. It makes the API just plain horrible to use. The good thing is that taichi never exposes the python bindings to the user, so we are good I guess.